### ISSUE TITLE
Add topics to repos

### ISSUE THREAD

**Issue by m1ga** [tidev/organization-docs#33]
So we can find them better in the github search (and maybe use it for gittio) we should add tags/topics to the repos.

for our modules:
`titanium-module`, `titanium-sdk`

for the repos (e.g. the sdk):
`titanium-sdk` and other like `mobile` `native` `android` `ios` like https://github.com/tidev/titanium_mobile if appropriate

**Comment by m1ga** [comment]
@hansemannn the issue about the github topics
Then we can scape e.g. https://github.com/topics/titanium-module (115 already)

**Comment by cb1kenobi** [comment]
I remember going through all the repos and making sure the topics were semi-accurate. It's not a bad way to get a list of modules.

**Comment by hansemannn** [comment]
Here is an example API request that returns all repos with the label "titanium" or "ios". We could fine-tune it as we develop it: https://api.github.com/search/repositories?q=titanium%20native%20in:topics&sort=created&order=asc

**Comment by hansemannn** [comment]
Let's goooo! https://github.com/tidev/module-search-www

**Comment by de1mat** [comment]
Random ideas.

Could have different levels of "verification". If module not updated for X years then show a warning. If module has tag for "ti-sdk-12" then can show it supports the latest SDK for eg.

Also show the number of stars the module repo has of course 😀

**Comment by m1ga** [comment]
some early feedback:

I think the results should not be a direct github api call. A cronjob should do the crawl and check the repo for "ios/manifest" or "android/manifest". That would remove many false positives. 

And add a weight to the title. If I search for "map" the actual ti.map is the 3rd result. First one is mixpanel, 2nd the sdk itself.

If you clear the complete input field it will break everything.

**Comment by hansemannn** [comment]
Yep, an own site aka gitt.io would be great, but also creates much more overhead to initially create. I wanted to get something out quickly, but of course if can be extended. Next.js also has the capability to do the server-side work directly in the same project (via the /api folder), so that could be a quick start.

**Comment by de1mat** [comment]
I am in favour of using Next.js as I think it is the tool of choice for the overall website and would have broader interest from other devs moving forward in terms of contributing etc. Also provides a lot of options around tooling and ecosystem.

**Comment by Astrovic** [comment]
Good idea! Maybe JavaScript could be added in the query, there are some CommonJS or Hyperloop modules.

**Comment by hansemannn** [comment]
It's all open source, go for it! :)
